docs/sources/operators-guide/getting-started-helm-charts/_index.md

```diff
@@ -0,0 +1,295 @@
+
+     ```yaml
+     server:
+       http_listen_port: 12345
+       grpc_listen_port: 54321
+
+     metrics:
+       wal_directory: /tmp/grafana-agent/wal
+
+       configs:
+         - name: agent
+           scrape_configs:
+             - job_name: agent
+               static_configs:
+                 - targets: ["127.0.0.1:12345"]
+           remote_write:
+             - url: http://<ingress-host>/api/v1/push
+     ```
+
+     In this case, your Grafana Agent writes metrics to Grafana Mimir that it scrapes from itself.
+
+  1. Create an empty directory for the write ahead log (WAL) of the Grafana Agent
+
+  1. Start a Grafana Agent by using Docker:
+
+     ```bash
+     docker run --network=host  -v <absolute-path-to-wal-directory>:/etc/agent/data -v <absolute-path-to>/agent.yaml:/etc/agent/agent.yaml grafana/agent
+     ```
+
+## Query metrics in Grafana
+
+First install Grafana, and then add Mimir as a Prometheus data source.
+
+1. Start Grafana by using Docker:
+
+   ```bash
+   docker run --rm --name=grafana --network=host grafana/grafana
+   ```
+
+1. In a browser, go to the Grafana server at [http://localhost:3000](http://localhost:3000).
+1. Sign in using the default username `admin` and password `admin`.
+1. On the left-hand side, go to **Configuration** > **Data sources**.
+1. Configure a new Prometheus data source to query the local Grafana Mimir cluster, by using the following settings:
+
+   | Field | Value                              |
+   | ----- | ---------------------------------- |
+   | Name  | Mimir                              |
+   | URL   | http://\<ingress-host\>/prometheus |
+
+   To add a data source, see [Add a data source](https://grafana.com/docs/grafana/latest/datasources/add-a-data-source/).
+
+1. Verify success:
+
+   You should be able to query metrics in [Grafana Explore](http://localhost:3000/explore),
+   as well as create dashboard panels by using your newly configured `Mimir` data source.
+
+## Set up metamonitoring
+
+Grafana Mimir metamonitoring collects metrics or logs, or both,
+about Grafana Mimir itself.
+In the example that follows, metamonitoring scrapes metrics about
+Grafana Mimir itself, and then writes those metrics to the same Grafana Mimir instance.
+
+1. To enable metamonitoring in Grafana Mimir, add the following YAML snippet to your Grafana Mimir `custom.yaml` file:
+
+   ```yaml
+   metaMonitoring:
+     serviceMonitor:
+       enabled: true
+     grafanaAgent:
+       enabled: true
+       installOperator: true
+       metrics:
+         additionalRemoteWriteConfigs:
+           - url: "http://mimir-nginx.mimir-test.svc:80/api/v1/push"
+   ```
+
+1. Upgrade Grafana Mimir by using the `helm` command:
+
+   ```bash
+   helm -n mimir-test upgrade mimir grafana/mimir-distributed -f custom.yaml
+   ```
+
+1. From [Grafana Explore](http://localhost:3000/explore), verify that your metrics are being scraped, by querying `sum(rate(cortex_ingester_ingested_samples_total[$__rate_interval]))`.
```

Comment:
Metrics can still be getting scraped, but this query can return no data or `0` if remote-write isn't set up correctly.

```suggestion
1. From [Grafana Explore](http://localhost:3000/explore), verify that your metrics are being written to Grafana Mimir, by querying `sum(rate(cortex_ingester_ingested_samples_total[$__rate_interval]))`.
```

Reply:
added